transform/transformers/MWSSTransformer.py

```diff
@@ -122,18 +121,11 @@ def transform(data, survey=None):
 
 
 def main(args):
-    Settings = namedtuple(
-        "Settings",
-        [
-            "FTP_HOST",
-            "SDX_FTP_IMAGE_PATH",
-        ]
-    )
 
     reply = json.load(args.input)
     transformer = MWSSTransformer(reply, seq_nr=args.seq_nr)
     zipfile = transformer.pack(
-        settings=Settings("\\\\NP3RVWAPXX370\\SDX_Prod\\", "EDC_QImages"),
+
```

Comment:
Is there a reason why the line is empty here?  Also do we not need the settings anymore?

Reply:
It's not needed anymore, the reason it was being passed was that the ImageTransformer in common couldn't import settings from the sub module. So this named tuple was a workaround with prod config in it :(

MWSS is now using the same ImageTransformer as other services so it gets its settings from the environment variables using this line:
image_path = settings.FTP_HOST + settings.SDX_FTP_IMAGE_PATH + "\\Images"

Reply:
:+1: